source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.WinUI.Wasm/SkiaSharp.Views.Uno.WinUI.Wasm.csproj

```
@@ -18,9 +20,11 @@
     <ProjectReference Include="..\..\..\binding\SkiaSharp\SkiaSharp.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <None Remove="nuget\build\netstandard2.0\SkiaSharp.Views.Uno.WinUI.targets" />
     <None Remove="WasmScripts\SkiaSharp.Views.Uno.Wasm.js" />
+    <None Remove="nuget\build\netstandard2.0\SkiaSharp.Views.Uno.WinUI.targets" />
     <None Include="nuget\build\netstandard2.0\SkiaSharp.Views.Uno.WinUI.targets" Link="nuget\build\$(PackagingPlatform)\SkiaSharp.Views.Uno.WinUI.targets" />
+    <None Remove="nuget\build\net7.0\SkiaSharp.Views.Uno.WinUI.targets" />
+    <None Include="nuget\build\net7.0\SkiaSharp.Views.Uno.WinUI.targets" Link="nuget\build\$(PackagingPlatform)\SkiaSharp.Views.Uno.WinUI.targets" />
```

I think you have a targets for each TFM, so maybe you just need to do this to add the Link attribute?

```suggestion
    <None Update="nuget\build\$(PackagingPlatform)\SkiaSharp.Views.Uno.WinUI.targets" Link="nuget\build\$(PackagingPlatform)\SkiaSharp.Views.Uno.WinUI.targets" />
```